bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -307,6 +307,7 @@ spec:
                 - --enable-leader-election
                 - --zap-log-level=info
                 - --zap-time-encoding=rfc3339nano
+                - --feature-gates=+operator.autoinstrumentation.go
```

Comment:
Should we keep this disabled here initially given this is an alpha feature?

Reply:
We definitely want Go instrumentation disabled by default.  I think this is getting changed because I had to change `config/default/manager_auth_proxy_patch.yaml` to ensure the feature flag was enabled for e2e tests.  Is there a way I can keep the Go e2e test and not mess with the manifest?  If the e2e test is using our default manifest we'll have to remove the Go e2e tests for now.

Reply:
I see... hmm @pavolloffay thoughts on what to do here?

Reply:
Based on the SIG meeting it sounded like it is proper for the default manifest to expose all available features.  If that is still the sentiment keeping this flag is appropriate.